mlir/test/fusion/pr-e2e/mixr-sd-explicit-broadcasting.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: rocmlir-driver -kernel-pipeline=migraphx %s | rocmlir-driver -host-pipeline=partition,highlevel -targets %arch | rocmlir-gen -ph -rand 1 -rand_type float -fut mlir_reshape_convolution --verifier clone - | rocmlir-driver -c -arch %arch | mlir-cpu-runner -O2 --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_float16_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext --entry-point-result=void | FileCheck %s
+// ALLOW_RETRIES: 2
+// CHECK: [1 1 1]
+// Note the fake wrapper function to make it look like this has already been partitioned,
+// since we want "partition" for the xmodel packaging but don't actually want to use
+// the partition logic.
+// This is an awkward hack that should be fixed Later (tm)
+module {
+  func.func @mlir_reshape_convolution_real(%arg0: tensor<1x1x16x1x16x1xf32>, %arg1: tensor<1x1x3x3xf32>) -> tensor<1x1x32x32xf32> attributes {kernel = "mixr", num_cu = 48 : i64} {
```

Comment:
This test isn;t doing anything meaningful as I have found out.
So are the e2e  tests in https://github.com/ROCmSoftwarePlatform/rocMLIR/pull/1222

Reply:
Really? Can we sync up on this - I looked at the intermediates and saw an actual series of calls.

Reply:
Ive put a reproducer on your past PR. 
Would you do me a favor and double check the output after rocmlir-driver -c just before cpu-runner.

I dont see any kernels there.

Reply:
i.e. no long bitstream anywhere to be found.